### PR TITLE
update README

### PR DESCRIPTION
**Pull request by AndyHWChung**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,48 +1,79 @@
...
-| -             | -             | anything else             | return found                |      
+The table below describes responses to a number of API's. The journeys for individuals that you will be able to trigger on the applicant
+frontend service will be rooted in the national insurance number (NINO) and the self assessment unique tax reference 
+(SA UTR) if an SA journey is chosen. The columns should be interpreted like so:
+- The `Date of Birth` and `SA UTR` columns are the values that will be returned by the citizen details API stub for the
+  given NINO. The SA UTR is an optional field in the response and it is not always returned by the stub. The name that is
+  returned by the stub is always the same
+- the `SA status` column describes the stub responses to the SA return status API. The input to this API is an SA UTR 
+
+If using a stubbed login, to be able to use an SA UTR which is different thn `1234567895` use an `NS...` NINO in the
```
Comment (LokeshSrinivasa) [inline README.md:56]: reword then,

File: README.md [hunk 2]
```diff
@@ -1,48 +1,79 @@
...
+frontend service will be rooted in the company registration number (CRN). The columns should be interpreted like so:
+- The `Company Name` column describes the response given by the companies house API. Given a particular
+  CRN, either a successful response with the given company name will be returned or an error response will be issued
+- The `CT UTR` column describes the response given by the DES "Get CT reference" API. This represents what HMRC records
+  store as the corporation tax unique tax identifier (CT UTR) for a given CRN. It is used to match a CRN with a CT UTR
+- The `CT status` columns describes the response to the CT return status API. The input to the API is the CT UTR of the  
+  company
+ 
+If using a stubbed login, the CT UTR can either be given as an enrolment or entered in as part of the applicant frontend
+journey if a CT enrolment has not been given with the CT UTR which matches the CRN given in the journey.
```
Comment (LokeshSrinivasa) [inline README.md:21]: statement is complete?

File: README.md (hunk 1 above)
Comment (AndyHWChung) [reply README.md]: updated - any better?

File: README.md (hunk 2 above)
Comment (LokeshSrinivasa) [reply README.md]: fine

File: README.md (hunk 1 above)
Comment (LokeshSrinivasa) [reply README.md]: i see 'thn' , please change 'then'

Comment (AndyHWChung) [reply README.md]: whoops 😆  I meant `than` but apparently it's more grammatically correct to use `from` according to intellij

Comment (LokeshSrinivasa) [reply README.md]: nice

**Review by LokeshSrinivasa — APPROVED** [review]

**Pull request merged**